composer.json

```diff
@@ -25,6 +25,7 @@
         "symfony/form": "^4.4 || ^5.0",
         "symfony/http-foundation": "^4.4 || ^5.0",
         "symfony/http-kernel": "^4.4 || ^5.0",
+        "symfony/mime": "^4.4 || ^5.0",
```

Comment:
Do we need mime?

Reply:
Yes, a few calls to `File::getMimeType` like this:
https://github.com/dustin10/VichUploaderBundle/blob/2903b5647621c9b5b5fbaac38cb481f50fc7d192/src/Storage/AbstractStorage.php#L39
And because of this (see https://github.com/symfony/symfony/blob/5.1/UPGRADE-5.1.md#httpfoundation): 
> HttpFoundation: Made the Mime component an optional dependency